### ISSUE TITLE
Could it be possible to get 2 more Grid information 

### ISSUE THREAD

**Issue by Wind06** [dmslabsbr/hoymiles#148]
Hello,

I would like to know if 2 other data can be collected:

- To  and From Grid

This will be very helpfull for building statistics with HomeAssistant.
Below is a screen shot. Of course I will be please to help in case more information or testing requiered.

Thanks.

![Capture d’écran 2024-05-11 à 22 57 57](https://github.com/dmslabsbr/hoymiles/assets/64360875/ed2de413-3c58-4c38-8537-056f5492bdf6)

**Comment by Cosik** [comment]
@Wind06 but those values are already provided by edge version

**Comment by Wind06** [comment]
you are right, meter informations are povided. To avoid local calculation I was wondering if the "to Grid" and "From Grid" can be downloaded from the cloud ?

**Comment by Cosik** [comment]
But to grid and from grid are not reading from meter?

**Comment by Wind06** [comment]
what is read from grid is negative or positive value depending if From or To grid.
To Grid is positive value
From Grid is negative value
Value are reported in "sensor.hoymiles_gateway_solarh_775030_meter_grid_power"

![Capture d’écran 2024-05-12 à 10 22 04](https://github.com/dmslabsbr/hoymiles/assets/64360875/9e225063-032d-4307-93a8-2658c0048606)

**Comment by Cosik** [comment]
@Wind06 but there are "meter_to_grid" and "meter_from_grid" which are already provided and those are energy values. Those values could be used in HA energy dashboard and it will calculate all details.

**Comment by Wind06** [comment]
indeed yes, but the meter_from_grid is always at 0W and the from_grid do have an impressive value which is not the value I have in the hoymiles dashboard.

**Comment by Cosik** [comment]
Ok, so as I mentioned, I will need access.

**Comment by Cosik** [comment]
I checked and found that probably there is a bug in cloud data getting. Addon is checking total data. On checked installation total data of return to grid value is 0. When I checked month by month return values to grid are non zero. In present implementation it is not possible to fix it. It will required changing type of meter data from constant increasing to something other. Also request has to be changed. 

Maybe in new release it will be possible.
